### ISSUE TITLE
A value of type 'Future<Image>' can't be assigned to a variable of type 'Image'.

### ISSUE THREAD

**Issue by sebasval** [EPNW/painter#8]
I have this error when download the project
A value of type 'Future<Image>' can't be assigned to a variable of type 'Image'. in the method of class painter.

In the line 166 to 168:

 Image toImage(){
    return picture.toImage(width, height); // here!
  }

**Comment by gbuela** [comment]
I'm getting the same :(

**Comment by kamalshree** [comment]
Same here unable to use the plugin below is the error message
```
 - 'Future' is from 'dart:async'.
 - 'Image' is from 'dart:ui'.
Try changing the type of the left hand side, or casting the right hand side to 'Image'.
    return picture.toImage(width, height);
                   ^
```

**Comment by freshteapot** [comment]
There is a fix for this in the pull request #5.

**Comment by EP-u-NW** [comment]
This was already solved by #5.
